models/classes/runner/map/QtiRunnerMap.php

```diff
@@ -324,12 +324,12 @@ protected function updateStats(&$target, $itemInfos)
 
         if (empty($itemInfos['informational'])) {
             $target['stats']['questions'] ++;
+
+			if (!empty($itemInfos['answered'])) {
+				$target['stats']['answered'] ++;
+			}
```

Comment:
Please be careful with indent while moving blocks of code. Messy code can lead to broken [window effect](https://en.wikipedia.org/wiki/Broken_windows_theory).